### ISSUE TITLE
Notify user that the event has been added to pending instead of confirming the event in the add event form

### ISSUE THREAD

**Issue by Caleb-Cohen** [Together-100Devs/Together#409]
**Related Issues**
Project: Admin Dashboard
Depends on: 
- #408 

**Description**
Event form confirmation message is changed to say event has been added to pending and will be reviewed before approval. 

**Acceptance Criteria**
Cypress test updated to expect pending message within the form.

**Comment by emmebravo** [comment]
@DevinCLane @intelagense was never worked on, is it still something we want?

**Comment by DevinCLane** [comment]
yeah goes in category of "what do we want to do with admin dashboard"

**Comment by DevinCLane** [comment]
closing as stale
